forge-gui/res/cardsfolder/upcoming/blade_of_shared_souls.txt

```diff
@@ -0,0 +1,10 @@
+Name:Blade of Shared Souls
+ManaCost:2 U
+Types:Artifact Equipment
+K:For Mirrodin
+T:Mode$ Attached | ValidSource$ Card.Self | Static$ True | TriggerZones$ Battlefield | Execute$ TrigCopy | TriggerDescription$ Whenever CARDNAME becomes attached to a creature, for as long as CARDNAME remains attached to it, you may have that creature become a copy of another target creature you control.
+SVar:TrigCopy:DB$ Clone | Choices$ Creature.YouCtrl+Other | CloneTarget$ Creature.EquippedBy | Duration$ UntilUnattached | SubAbility$ DBCleanup
```

Comment:
It needs to Target

Reply:
UntilUnattached needs some special logic in the CloneEffect, but that can be done later